lib/matplotlib/backends/backend_ps.py

```diff
@@ -536,7 +536,10 @@ def draw_markers(
         ps_cmd = ['/o {', 'gsave', 'newpath', 'translate'] # don't want the translate to be global
 
         lw = gc.get_linewidth()
-        stroke = lw != 0.0
+        alpha = (gc.get_alpha()
```

Comment:
This effectively treats alpha as a bool for ps?

Reply:
We do the same for text.